app/connector/aws-ddb/pom.xml

```diff
@@ -92,6 +92,10 @@
       <groupId>io.syndesis.connector</groupId>
       <artifactId>connector-support-util</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Please see my comment below.  Adding it to a `pom.xml` seems cheap, but we have to maintain these libraries for a long time. I would recommend removing this, unless there is a really good reason for intriducing commons-lang.

Reply:
Ok!